core-server/src/main/java/org/glassfish/jersey/server/model/IntrospectionModeller.java

```diff
@@ -208,8 +210,11 @@ private void checkResourceClassFields(final boolean encodedFlag,
                         field.getGenericType(),
                         field.getAnnotations());
                 if (null != p) {
```

Comment:
why not to extend existing IF by adding another condition? (same question is valid for the first modification in that class)